questionable.gemspec

```diff
@@ -1,33 +1,31 @@
-$:.push File.expand_path("../lib", __FILE__)
+$:.push File.expand_path('../lib', __FILE__)
 
-# Maintain your gem's version:
-require "questionable/version"
+require 'questionable/version'
 
-# Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name        = "questionable_surveys"
+  s.name        = 'questionable_surveys'
   s.version     = Questionable::VERSION
```

Comment:
Do you think it's worth adding a patch version for this? I think it's fine given nothing really changed functionally...